frame/contracts/src/lib.rs

```diff
@@ -562,6 +562,8 @@ pub mod pallet {
 	pub enum Error<T> {
 		/// A new schedule must have a greater version than the current one.
 		InvalidScheduleVersion,
+		/// Invalid combination of flags supplied to `seal_[delegate]_call`.
```

Comment:
```suggestion
		/// Invalid combination of flags supplied to `seal_delegate_call`.
```

Reply:
It relates to both `seal_call` and `seal_delegate_call`. Maybe not the best syntax though.

Reply:
Can you change the comment to convey that intention then? Something like `seal_call or seal_delegate_call` will do

Reply:
Sure